src/server/Server.ts

```diff
@@ -16,8 +16,23 @@ import Logger from '@/utils/Logger';
 import CommandManager from '@/command/CommandManager';
 import RakNet from '@/network/rakNet/RakNet';
 import PacketManager from '@/network/bedrock/PacketManager';
+import { ServerOptions } from './ServerOptions'
 
-class Server extends EventEmitter {
+class Server extends EventEmitter implements ServerOptions {
+    ServerName = "NetrexTestServer"
+    Server_IP = "127.0.0.1" // If you are running the server on your local machine, please change the server ip to SERVER_IP
+    Port = 19132 // By Default this will be 19132
+    MaxPlayers = 5; // Only change this the recommended limit that your host suggests
+    MOTD = "Test Server : 1.14.6"
+    DefaultGamemode = 0; // 0 for survival, 1 for creative
+    EnableQuery = true; //true for yes, false for no
+    EnableRCON = true; //true for yes, false for no
+    RCONPassword = "Olybear9IsStinky";
```

Comment:
Please refrain from writing this in pull requests

Reply:
k